.github/workflows/CD.yml

```diff
@@ -505,9 +505,9 @@ jobs:
           retry_wait_seconds: 20
           command: cd testing-framework/terraform/eks && terraform destroy -auto-approve
 
-  delete-images-binaries-if-fail:
+  delete-images-binaries:
     runs-on: ubuntu-latest
-    if: failure()
+    if: $ {{ failure() || cancelled() }}
```

Comment:
```suggestion
    if: ${{ failure() || cancelled() }}
```

Reply:
Nice catch. Fixed it in the newest commit.